### DISCUSSION TITLE
How to close console window?

### DISCUSSION THREAD

**Discussion by MouriKogorou** [can-lehmann/owlkettle#159] [Q&A] [answered]
hello, Is there hava any way to close console window?
![image](https://github.com/can-lehmann/owlkettle/assets/43428806/d1f5b949-7f37-49db-bc02-4338f0512ef3)

**Comment by can-lehmann** [comment, marked as answer]
Passing `--app:gui` to the nim compiler should hide it. See https://nim-lang.org/docs/nimc.html
